hieradata/site/ls/role/bastion.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 classes:
   - "profile::core::nfsclient"
+  - "profile::it::icinga_agent"
```

Comment:
The first approach was adding it there and creating an "uncommon" profile, so icinga-master could use. But as I stated in my first comment, if we add new roles, an they don't have a reflection of the icinga-agent parameters in the private repo, it will crash.

Reply:
it will be fewer lines of code to exclude `profile::core::icinga_agent` from just the `icinga_master` rather than than explicitly adding it to every other role.  E.g.

```
diff --git a/site/profile/manifests/core/common.pp b/site/profile/manifests/core/common.pp
index 95a984d1..1e0b93a1 100644
--- a/site/profile/manifests/core/common.pp
+++ b/site/profile/manifests/core/common.pp
@@ -5,7 +5,9 @@
 #   Enable or disable metrics collection. Metrics collection may be disabled on development
 #   nodes, nodes that don't have uptime requirements, or nodes that should only have minimal
 #   software load.
-class profile::core::common {
+class profile::core::common(
+  Boolean $icinga_agent = true,
+) {
   include timezone
   include tuned
   include chrony
@@ -25,4 +27,8 @@ class profile::core::common {
   include rsyslog::config
   include profile::core::hardware
   include profile::core::dielibwrapdie
+
+  if $icinga_agent {
+    include profile::core::icinga_agent
+  }
 }
```

And then adding `profile::core::common::icinga_agent: false` to the `icinga_master` role.

Reply:
Done as suggested